docs/release-process.md

```diff
@@ -0,0 +1,309 @@
+```
+
+
+Create binary promotion PR:
+
+```
+cd ${GOPATH}/src/k8s.io/k8s.io
+
+git co master
+git co -b kops_artifacts_${VERSION}
+
+mkdir -p ./k8s-staging-kops/kops/releases/${VERSION}/
+gsutil rsync -r  gs://k8s-staging-kops/kops/releases/${VERSION}/ ./k8s-staging-kops/kops/releases/${VERSION}/
+
+promobot-generate-manifest --src k8s-staging-kops/kops/releases/ >> artifacts/manifests/k8s-staging-kops/${VERSION}.yaml
+```
+
+Verify, then send a PR:
+
+```
+git add artifacts/manifests/k8s-staging-kops/${VERSION}.yaml
+git commit -m "Promote kops $VERSION binary artifacts"
+git push ${USER}
+hub pull-request
+```
+
+
+## Promote to dockerhub / S3 / github (legacy)
+
+We are in the process of moving to k8s.gcr.io for all images and to
+artifacts.k8s.io for all non-image artifacts.
+
+In the meantime (and for compatibility), we must also promote to the old locations:
+
+Images to dockerhub:
+
+```
+crane cp gcr.io/k8s-staging-kops/kube-apiserver-healthcheck:${VERSION} kope/kube-apiserver-healthcheck:${VERSION}
+crane cp gcr.io/k8s-staging-kops/dns-controller:${VERSION} kope/dns-controller:${VERSION}
+crane cp gcr.io/k8s-staging-kops/kops-controller:${VERSION} kope/kops-controller:${VERSION}
+```
+
+
+Binaries to S3 bucket:
+
+```
+aws s3 sync --acl public-read k8s-staging-kops/kops/releases/${VERSION}/ s3://kubeupv2/kops/${VERSION}/
+```
+
+Binaries to github:
+
+```
+cd ${GOPATH}/src/k8s.io/kops/
+shipbot -tag v${VERSION} -config .shipbot.yaml -src ${GOPATH}/src/k8s.io/k8s.io/k8s-staging-kops/kops/releases/${VERSION}/
+```
+
+
+Until the binary promoter is automatic, we also need to promote the binary artifacts manually:
+
+```
+mkdir -p /tmp/thin/artifacts/filestores/k8s-staging-kops/
+mkdir -p /tmp/thin/artifacts/manifests/k8s-staging-kops/
+
+cd ${GOPATH}/src/k8s.io/k8s.io
+cp artifacts/manifests/k8s-staging-kops/${VERSION}.yaml /tmp/thin/artifacts/manifests/k8s-staging-kops/
+
+cat > /tmp/thin/artifacts/filestores/k8s-staging-kops/filepromoter-manifest.yaml << EOF
+filestores:
+- base: gs://k8s-staging-kops/kops/releases/
+  src: true
+- base: gs://k8s-artifacts-prod/binaries/kops/
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+EOF
+
+promobot-files --filestores /tmp/thin/artifacts/filestores/k8s-staging-kops/filepromoter-manifest.yaml --files /tmp/thin/artifacts/manifests/k8s-staging-kops/ --dry-run=true
+```
+
+After validation of the dry-run output:
+```
+promobot-files --filestores /tmp/thin/artifacts/filestores/k8s-staging-kops/filepromoter-manifest.yaml --files /tmp/thin/artifacts/manifests/k8s-staging-kops/ --dry-run=false --use-service-account
+```
+
+## Smoketesting the release
+
+```
+wget https://artifacts.k8s.io/binaries/kops/${VERSION}/linux/amd64/kops
+
+mv kops ko
+chmod +x ko
+
+ko version
+```
+
+Also run through a kops create cluster flow, ideally verifying that
+everything is pulling from the new locations.
+
+## On github
+
+* Download release
+* Validate it
+* Add notes
```

Comment:
I think we should just link to notes on github. The links break since we expect people to view them through kops.sigs.k8s.io.